### DISCUSSION TITLE
Can the Micro Bolus Icon threshold be customized

### DISCUSSION THREAD

**Discussion by alexanderjr11** [NightscoutFoundation/xDrip#2180] [Q&A]
HI I was curious if there was a way to configure the micro bolus threshold to be higher than the .1U threshold set now. Ideally for my use case I would like to set it to anything .5U and below Thanks

**Comment by Navid200** [comment]
Icon?
I'm not sure what you mean.

Is the following by any chance relevant?
https://github.com/NightscoutFoundation/xDrip/issues/1878

**Reply by alexanderjr11** [reply]
I noticed that issue as well when testing. but no my question is slightly different. if you go to Settings>Display Settings>Graph Settings there is the option to "Show Micro Bolus Icons" in an attempt to declutter the graph. It appears from my testing and playing around that only a .1U bolus will show the micro bolus icon. (Side Note: This is where I noticed the issue in [#1878](https://github.com/NightscoutFoundation/xDrip/issues/1878) I was only able to enter .1U and if I put anything less like .05U or .09U it just defualts back to .1U ) 

My ask was to see if I could configure the value of what xDrip considers to be a micro bolus. i.e. If I have high insulin resistance and a micro bolus for me might be .5U where as for someone else it might be that .1U threshold or potentially even higher than .5U


Does that make sense? If needed I can get the screenshots of the menu items Im talking about. Thanks for the quick response!

**Comment by Navid200** [comment]
@tolot27 Do you think this is a bug?  
Do we need an issue?

**Reply by tolot27** [reply]
> @tolot27 Do you think this is a bug? Do we need an issue?

This is more of a feature request than a bug. The bug #1878 will be fixed with #2186. But to make super-microbolus (SMB) threshold configurable we would need another preference setting.

The [current](https://github.com/NightscoutFoundation/xDrip/blob/0f4ad45e0d96794dd71619c6ea63768847f807e1/app/src/main/java/com/eveningoutpost/dexdrip/Models/Treatments.java#L1294) SMB threshold is 0.3 and 0.4 for OpenAPS which should be sufficient [Introducing oref1 and super-microboluses (SMB)](see https://diyps.org/2017/04/30/introducing-oref1-and-super-microboluses-smb-and-what-it-means-compared-to-oref0-the-original-openaps-algorithm).
